test/integration/xpack-stig.yml

```diff
@@ -0,0 +1,198 @@
+---
+# STIG based xpack provision for inspec hardening compliance
+# Make sure to change the variables below in the vars section
+# and in the kitchen.vagrant.yml file
+
+- import_playbook: roles/elasticsearch/test/integration/xpack.yml
```

Comment:
This isn't working for me when I run it locally. The path is relative to this file so I think it needs to be something like `elasticsearch/test/integration/xpack.yml`

```
     ERROR! Unable to retrieve file contents
       Could not find or access '/tmp/kitchen/roles/elasticsearch/test/integration/xpack.yml'
>>>>>> ------Exception-------
>>>>>> Class: Kitchen::ActionFailed
>>>>>> Message: 1 actions failed.
>>>>>>     Converge failed on instance <xpack-stig-centos-7>.  Please see .kitchen/logs/xpack-stig-centos-7.log for more details
>>>>>> ----------------------
>>>>>> Please see .kitchen/logs/kitchen.log for more details
>>>>>> Also try running `kitchen diagnose --all` for configuration
```